<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemform</class>
 <widget class="QWidget" name="itemform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>48</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cardID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>21</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>45</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>单元名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
